Gym Env Checker (#615)
hill-a/stable-baselines: docs/guide/rl_tips.rst

- debug with random actions to check that your environment works and follows the gym interface:

> We provide an helper to check that your environment should run without error:

Reviewer: ```suggestion
We provide a helper to check that your environment runs without error:
```
